pyfar/utils.py

```diff
@@ -212,3 +212,45 @@ def concatenate_channels(signals, caxis=0, broadcasting=False):
         return pf.TimeData(data, signals[0].times)
     else:
         return pf.FrequencyData(data, signals[0].frequencies)
+
+
+def concatenate_bins(signals):
```

Comment:
Since we now have functions to broadcast cshapes of audio objects it would be nice to include this by default to make this more flexible.